React/React-Core.podspec

```diff
@@ -43,7 +43,8 @@ Pod::Spec.new do |s|
                              "UIUtils/*",
                              "Profiler/{RCTFPSGraph,RCTPerfMonitor}.*",
                              "Profiler/RCTProfileTrampoline-{arm,arm64,i386}.S",
-                             "Base/{RCTPlatform,RCTKeyCommands}.*",
```

Comment:
Is this another diff from fb's master we'd be taking on?

Reply:
Nope, these are already `macOS` specific.